source/includes/fact-older-version-limitations.rst

```diff
@@ -29,8 +29,8 @@
   enable the ``reversible`` option in the :ref:`/start <c2c-api-start>` request.
 
 - You can't enable the ``enableUserWriteBlocking`` option in the ``/start`` 
-  request. Ensure that no writes are made to the source or destination cluster 
-  during the migration.
+  request, so dual write-blocking is not supported. Destination-only 
```

Comment:
can we keep the note that users should ensure that no writes are made to the source cluster after commit has been issued?